pandas/core/window/rolling.py

```diff
@@ -2382,8 +2385,8 @@ def kurt(self, numeric_only: bool = False):
         create_section_header("Parameters"),
         dedent(
             """
-        quantile : float
```

Comment:
Please keep the old name here and add a deprecation note that this will get renamed to q

Reply:
No need to add the new description already and also the cols thing is unnecessary